multiview_mapping_toolkit/meshes/meshes.py

```diff
@@ -1012,9 +1012,9 @@ def get_vert_values_from_raster_file(
 
         # Set nodata locations to nan
         # TODO figure out if it will ever be a problem to take the first value
-        sampled_raster_values[
-            sampled_raster_values == raster.nodatavals[0]
-        ] = nodata_fill_value
+        sampled_raster_values[sampled_raster_values == raster.nodatavals[0]] = (
```

Comment:
It's fine to change the format for existing code, but I'm curious if you know what settings caused this change? 

Also, if you can figure out how to replicate this formatting locally (maybe using the `isort` profile for `black`) can you please include it in the `CONTRIBUTING.md`? This will help avoid extraneous formatting that is just switching between different configurations of `black`.